specification/ai/ImageAnalysis/main.tsp

```diff
@@ -14,7 +14,14 @@ using Azure.Core;
   title: "Image Analysis",
 })
 @versioned(ImageAnalysis.Versions)
-@useAuth(ApiKeyAuth<ApiKeyLocation.header, "Ocp-Apim-Subscription-Key">)
+@useAuth(ApiKeyAuth<ApiKeyLocation.header, "Ocp-Apim-Subscription-Key"> | OAuth2Auth<[
+  {
+    type: OAuth2FlowType.implicit,
```

Comment:
I see these possible values for OAuth2FlowType, but without a description. Do you know why you picked "implicit" here?

@doc("Describes the OAuth2 flow type")
enum OAuth2FlowType {
  @doc("authorization code flow")
  authorizationCode,

  @doc("implicit flow")
  implicit,

  @doc("password flow")
  password,

  @doc("client credential flow")
  clientCredentials,
}